Assets/Scripts/Player/PlayerStats.cs

```diff
@@ -93,7 +93,9 @@ public int currentDefence
     // Setter / getter for CurrentIncomingDamage
     public float CurrentIncomingDamage
     {
-        get { return CurrentIncomingDamage; }
+        // TODO look at replacement
+        //  get { return CurrentIncomingDamage; }
+        get { return IncomingDamage;}
```

Comment:
@joshualeecoss Please check this!

Reply:
This error should now be fixed on this branch. Sorry about that!